### PR TITLE
ref(cli): Refactor logging and Sentry initialization so they are handled consistently

### PR DESCRIPTION
**Pull request by tkaemming**

### CODE REVIEW

**Comment by tkaemming** [comment]
Something like #575 is a better solution to this, since that would ensure the `--log-level` is customizable for every command across the entire interface. That unfortunately is a breaking change.

The `cdc` command line interface is a good example of how to do this correctly: https://github.com/getsentry/cdc/blob/1f66b850812365d02b9a9a7942d5f10b4cbe4791/cdc/__main__.py#L23-L57

File: snuba/environment.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    logging.basicConfig(
+        level=getattr(logging, level, settings.LOG_LEVEL).upper(),
+        format="%(asctime)s %(message)s",
+    )
+
+
+def setup_sentry() -> None:
+    sentry_sdk.init(
+        dsn=settings.SENTRY_DSN,
+        integrations=[FlaskIntegration(), GnuBacktraceIntegration()],
```
Comment (fpacifici) [inline snuba/environment.py]: We likely don't need/want the FlaskIntegration for script (that are not running on flask). Does this hurt them ?

**Review by fpacifici — APPROVED** [review]
thanks, this is useful

File: snuba/environment.py (hunk 1 above)
Comment (tkaemming) [reply snuba/environment.py]: This is a good point that I had not considered. After a quick skim of the integration code, I don't think that there are any downsides as long as flask is importable: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/integrations/flask.py

**Pull request merged**